src/lightning/fabric/wrappers.py

```diff
@@ -199,3 +199,16 @@ def _unwrap(
         return obj
 
     return apply_to_collection(collection, dtype=(_FabricModule, _FabricOptimizer, _FabricDataLoader), function=_unwrap)
+
+
+def is_wrapped(obj: Union[Any, _FabricModule, _FabricOptimizer, _FabricDataLoader]) -> bool:
```

Comment:
"Any" makes mypy stop running: https://stackoverflow.com/a/39817126

```suggestion
def is_wrapped(obj: object) -> bool:
```